addon/components/ember-td/template.hbs

```diff
@@ -6,22 +6,23 @@
         data-test-select-row-container
       >
         {{-ember-table-private/simple-checkbox
```

Comment:
`<EmberTablePrivateSimpleCheckbox `

Reply:
Can't convert to angle brackets because of the 2.18 `expected an implementation for DynamicArg` issue